src/components/Gradebook/index.jsx

```diff
@@ -471,12 +471,28 @@ export default class Gradebook extends React.Component {
                   onClose={() => this.props.closeBanner()}
                   open={this.props.showSuccess}
                 />
-                <div>
-                  Showing
-                  <span className="font-weight-bold"> {this.props.filteredUsersCount} </span>
-                  of
-                  <span className="font-weight-bold"> {this.props.totalUsersCount} </span>
-                  total learners
+                <div className="d-flex justify-content-between mb-2">
```

Comment:
I had some trouble getting the alignment where I wanted it, so let me know if this doesn't make any sense. I am happy for some CSS help.

Reply:
I spoke to Matt, and this is actually a fine approach.

Reply:
I think `align-items-center` is what I was reaching for when we were talking through this in person: that would center the button and the explanatory text. If we want the header and the explanatory text as one thing, I think we'd want to use the same `justify-content` setting as you use here, and use `align-self` on the button to make it more centered relative to the stuff on the other side. And have the heading be inside the first child div of the one on this line.